pkg/controller/windowsmachine/metrics/metrics.go

```diff
@@ -0,0 +1,219 @@
+package metrics
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/operator-framework/operator-sdk/pkg/k8sutil"
+	"github.com/operator-framework/operator-sdk/pkg/metrics"
+	"github.com/pkg/errors"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/client-go/kubernetes"
+	k8sclient "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+
+	"github.com/openshift/windows-machine-config-operator/pkg/controller/windowsmachine/nodeconfig"
```

Comment:
Please group this correctly

Reply:
Not sure of the regrouping here . I have regrouped as -

First group : go packages
Second group : external repo
third group : wmco

Reply:
OK, I thought the second group was `wmco`. We need to add this grouping to the contribution guide.